src/static.h

```diff
@@ -11,6 +12,11 @@ typedef struct {
   bool extended;
 } Frame;
 
+typedef struct {
```

Comment:
```c
typedef struct {
  time_t stamp;
  Frame frame;
} TimestampedFrame;
```
might be more suitable.
The reordering is to minimize padding. The .stamp is inspired by [ROS](http://docs.ros.org/melodic/api/std_msgs/html/msg/Header.html). The renaming is cos you're mixing snake and camel case.